src/prefect/cli/kv_store.py

```diff
@@ -0,0 +1,96 @@
+import sys
+import click
+
+from prefect import config
+from prefect.backend import kv_store
+from prefect.cli.build_register import log_exception
+
+
+@click.group()
+def kv():
+    """
+    Interact with Prefect Cloud KV Store
+
+    \b
+    Usage:
+        $ prefect kv [COMMAND]
+    """
+    if config.backend == "server":
+        click.secho("Key value commands are only available with Prefect Cloud")
+        sys.exit(1)
+
+
+@kv.command(name="set")
+@click.argument("key")
+@click.argument("value")
+def set_command(key, value):
+    """
+    Set a key value pair, overriding existing values if key exists
+
+    \b
+    Arguments:
+        key         TEXT    Key to set
+        value       TEXT    Value associated with key to set
+    """
+    try:
+        kv_store.set_key_value(key=key, value=value)
+        click.secho("Key value pair set successfully", fg="green")
+    except Exception as exc:
+        log_exception(exc)
+        click.secho("An error occurred setting the key value pair", fg="red")
+        sys.exit(1)
+
+
+@kv.command(name="get")
+@click.argument("key")
+def get_command(key):
+    """
+    Get the value of a key
+
+    \b
+    Arguments:
+        key         TEXT    Key to get
+    """
+    try:
+        result = kv_store.get_key_value(key=key)
+        click.secho(f"Key {key} has value {result}", fg="green")
+    except Exception as exc:
+        log_exception(exc)
+        click.secho(f"Error retrieving value for key {key}", fg="red")
+        sys.exit(1)
```

Comment:
Let's import `TerminalError` and `handle_terminal_error` and use that instead since it's used in `build_register` and `run` it'll make it easier to refactor later

Reply:
done